src/main/game/HumanPlayer.java

```diff
@@ -0,0 +1,22 @@
+package main.game;
+
+public class HumanPlayer implements Player {
+
+    private final UI inputOutput;
```

Comment:
I would suggest keeping the naming consistent in that you have `UI` being the type so I would name the variable the same instead of `inputOutput`.